test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -83,6 +93,31 @@ def test_https_proxy(self) -> None:
             r = https.request("GET", f"{self.http_url}/")
             assert r.status == 200
 
+    def test_is_verified_http_proxy_to_http_target(self) -> None:
+        with proxy_from_url(self.proxy_url, ca_certs=DEFAULT_CA) as http:
+            r = http.request("GET", f"{self.http_url}/")
+            assert r.status == 200
+            assert_is_verified(http, False, False)
+
+    def test_is_verified_http_proxy_to_https_target(self) -> None:
+        with proxy_from_url(self.proxy_url, ca_certs=DEFAULT_CA) as http:
+            r = http.request("GET", f"{self.https_url}/")
+            assert r.status == 200
+            assert_is_verified(http, False, True)
+
+    @pytest.mark.skip(reason="see https://github.com/urllib3/urllib3/issues/3267")
```

Comment:
This should be `xfail` instead of `skip` so it starts failing when the test begins passing.